apps/easypid/src/features/pid/PidReviewRequestSlide.tsx

```diff
@@ -0,0 +1,37 @@
+import { usePidDisplay } from '@easypid/hooks'
+import { Button, Circle, Heading, HeroIcons, Paragraph, Spinner, Stack, YStack } from '@package/ui'
+import { CardWithAttributes, DualResponseButtons, useWizard } from 'packages/app/src'
+import { sanitizeString } from 'packages/utils/src'
+import { useState } from 'react'
+import { OnboardingIdCardRequestedAttributes } from '../onboarding/screens/id-card-requested-attributes'
+
+interface PidReviewRequestSlideProps {
+  title: string
+  requestedAttributes: string[]
+}
+
+export function PidReviewRequestSlide({ title, requestedAttributes }: PidReviewRequestSlideProps) {
+  const { onNext } = useWizard()
+  const display = usePidDisplay()
+
+  const [isLoading, setIsLoading] = useState(false)
+
+  const onContinue = () => {
+    if (isLoading) return
+    setIsLoading(true)
+
+    // Do logic
```

Comment:
what do you mean with do logic? shouldn't we do the logic here? isn't it implemented yet?

Reply:
This was placeholder, removed it.